### PR TITLE
Document NBXplorer stuck on pruned block

### PR DESCRIPTION
**Pull request by NicolasDorier**
This commit includes troubleshooting instructions to address an issue where NBXplorer may become stuck and fail to sync due to pruned blocks on a Bitcoin full node.

The recommended solution involves updating NBXplorer and, if the issue persists, forcing NBXplorer to skip the affected blocks. 

While this may cause some transactions to be missed, the BTCPay Server will be back online and operational.

### CODE REVIEW

**Comment by NicolasDorier** [comment]
I am waiting for somebody impacted by this problem to test the instructions.

File: docs/FAQ/General.md [hunk 1]
```diff
@@ -183,3 +183,28 @@ It's possible to [create backups of your BTCPay Server](https://docs.btcpayserve
 
 Currently charging users for using your BTCPay Server instance, whether it's a percentage of transactions or a registration fee, is not supported natively.
 Enabling such a feature could be possible, using the [Transmuter](https://docs.btcpayserver.org/Transmuter/) and/or the [Greenfield API](https://docs.btcpayserver.org/API/Greenfield/v1/) but will require moderate to extensive technical knowledge.
+
+## Stuck synching: "NBXplorer is synchronizing"
+
+In some cases, you may encounter NBXplorer getting stuck. The first thing to try in such a situation is to update it. If you are using the docker deployment, simply run `./btcpay-update.sh` or go to `Server settings / Maintenance / Update`.
```
Comment (dennisreimann) [inline docs/FAQ/General.md:189]:
```suggestion
In some cases, you may encounter NBXplorer getting stuck. The first thing to try in such a situation is to update it. If you are using the Docker deployment, simply run `./btcpay-update.sh` or go to `Server settings / Maintenance / Update`.
```

File: docs/FAQ/General.md [hunk 2]
```diff
@@ -183,3 +183,28 @@ It's possible to [create backups of your BTCPay Server](https://docs.btcpayserve
...
+
+If, despite restarting, the problem persists and NBXplorer remains stuck, you may notice that the synch dialog appears as shown below, and the height does not change such as in this screenshot:
+
+![NBXplorer-stuck](./../img/NBXplorer-stuck.png "NBXplorer is synchronizing")
+
+This issue generally occurs when your server has been offline for an extended period, and your Bitcoin full node is pruned, which is the default setting in BTCPay Server docker deployments.
```
Comment (dennisreimann) [inline docs/FAQ/General.md:195]:
```suggestion
This issue generally occurs when your server has been offline for an extended period, and your Bitcoin full node is pruned, which is the default setting in BTCPay Server Docker deployments.
```

**Review by dennisreimann — APPROVED** [review]

**Comment by NicolasDorier** [comment]
Got feedback that it worked for someone.

**Pull request merged**